packages/synthchron-simulator/src/process-engines/petrinet-engine.ts

```diff
@@ -12,7 +12,7 @@ const isAccepting: IsAcceptingType<ProcessModel, State, ActivityIdentifier> = (m
         // We only care about the places
         .filter((node): node is PetriNetPlace => node.type === "place")
         // Every place that has a token has to be accepting
-        .every(place => state.get(place.identifier) || 0 >= 1 === place.accepting)
+        .every(place => state.get(place.identifier) === undefined ? 0 : place.accepting(state.get(place.identifier)!))
```

Comment:
This is false. If we have a place that accepts when it has 0 tokens, and we have a state that has some place not defined, this will not accept